xoxzo/cloudpy/cloudpy.py

```diff
@@ -0,0 +1,199 @@
+# xoxzo api libirary
+
+'''
+This is the API library for Xoxzo's telephony service.
+http://docs.xoxzo.com/en/
+
+see http://docs.python-requests.org/en/master/ for requests library.
+'''
+
+import requests
+import os
+import json
+
+__author__ = "Akira Nonaka <nonaka@mac.com>"
+__status__ = "development"
+__version__ = "0.1"
+__date__ = "30 March 2011"
+
+
+class XoxzoResponse:
+
+    def __init__(self, message={}, messages=[], errors=None):
+        """
+
+        :rtype: XoxzoResponse
+        """
+        self.message = message
+        self.messages = messages
+        self.errors = errors
+
+
+class XoxzoClient:
+    '''
+    Base class to access Xoxzo API.
+
+    :param string sid: Your sid of the xoxzo account.
+    :param string auth_token: Your auth_token of the xoxzo account.
+    :param string api_host: If None, stadard xoxzo server will be used.
+    '''
+    REQUESTS_EXCEPITON = 499
```

Comment:
This is the value set in XoxzoResponse.errors when cloudpy  catches the Requests exception.